### DISCUSSION TITLE
Plotting negative elevation angles for RHI scans

### DISCUSSION THREAD

**Discussion by Ted-Whittock** [ARM-DOE/pyart#1253] [Q&A]
Hello,

I have a bunch of RHI files from a mobile radar that was positioned along a mountain ridge, and so we have negative elevation angles to capture the precipitation over the surrounding lower terrain. (Iink to sample data: https://drive.google.com/drive/folders/19cQ3aS5qBt2gsQYjZu5cg14U_j7qt5J8?usp=sharing).

I have stitched 2 different azimuths together manually and added terrain heights below the radar (see attached image). However, I cannot figure out why it appears that the negative elevation angles (down to -1.4 deg) are not plotting. Is there a way to plot these negative elevations, or does Pyart always ignore them?

I am also aware that the y-axis on my attached plot is incorrect, but I wanted to address the issue of the negative elevation angles first.

Thanks,
Ted

![RHI_3panel_2angle_DBZHCandZDRCandCC_20170203_160235_to_20170203_160317_Azimuth65_and_245](https://user-images.githubusercontent.com/107871515/187554396-746a4759-7f27-408a-b3fd-c96d39d2151e.png)

**Comment by scollis** [comment]
Great point @Ted-Whittock ! And thank you for providing the sample data. A good number of the Dev team are at ERAD this week but this is something we need to fix. Probably based on an assumption of the original Py-ART dev (me :) ) that such a case would not exist :) 
Amazing data BTW.

**Reply by Ted-Whittock** [reply]
Thanks so much for the quick reply! I thought that might be the case. Besides this one hangup, the RHI examples in the example gallery have been very helpful. Glad I didn't have to do all the calculations on my own to account for earth's curvature, etc!

**Reply by Ted-Whittock** [reply]
Hi @scollis, wondering if you and the team have had a chance to take a look at this as of yet.

-Ted
